README.md

```diff
@@ -34,8 +34,39 @@ GOOGLE_PRIVATE_KEY_ID=privatekeyid
 ```
 
 ### Important Notes
+
 - Make sure you have a valid <code>GOOGLE_PRIVATE_KEY</code> and <code>GOOGLE_PRIVATE_KEY_ID</code> specified in the .env. The base64 decoder parses these keys and throws error if invalid.
 
+### Updating environment variable names
```

Comment:
what about the values? how do we change those? 
Scenarios when we need to update env variable values will be more often than the names.